opentelemetry/src/global/error_handler.rs

```diff
@@ -18,12 +18,15 @@ pub enum Error {
     #[cfg(feature = "trace")]
     #[cfg_attr(docsrs, doc(cfg(feature = "trace")))]
     #[error(transparent)]
+    /// Failed to export traces.
     Trace(#[from] TraceError),
     #[cfg(feature = "metrics")]
     #[cfg_attr(docsrs, doc(cfg(feature = "metrics")))]
     #[error(transparent)]
+    /// Failed to export metrics.
```

Comment:
I think [`MetricsError`](https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry/src/metrics/mod.rs#L44) covers more than export errors. It also covers some configuration errors so maybe rephrase it into something like `errors from metrics module`  would be more accurate.

Reply:
Updated 👍🏼